.github/workflows/test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - uses: actions/setup-java@v1
         if: steps.changes.outputs.android == 'true'
         with:
-          java-version: 8
+          java-version: 11
```

Comment:
I'm not 100% sure, but I _think_ API level 26 (our min supported version) **doesn't support any Java versions higher than 8**. Did you test this?

Reply:
Unfortunately I don't have access to older Andoid versions now (Mac M1 etc), but I understand there shouldn't be any problems as long as we set `minSdkVersion: 24` and everything still compiles? https://stackoverflow.com/questions/54129834/which-android-versions-run-which-java-versions

Reply:
Lemme just quickly try this one locally!

Reply:
Yeah, doesn't work :( Lemme post more details below.

Reply:
Testing version and the target version might be different, I guess. I've also done the same upgrade in #580. We might have no other way out that doing that eventually.